package.json

```diff
@@ -1,107 +1,40 @@
-    "analyze": "cross-env-shell NODE_OPTIONS=\"-r esm\" ANALYZE=true next build",
-    "lint": "prettier --write . --loglevel=warn && eslint . --quiet --fix",
-    "eslint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
-    "start": "cross-env NODE_OPTIONS=\"-r esm\" next start"
-  },
-  "dependencies": {
-    "@next/bundle-analyzer": "^10.0.6",
-    "@reduxjs/toolkit": "^1.5.0",
-    "@svgr/webpack": "^5.5.0",
-    "@types/lodash": "^4.14.168",
-    "@types/react-redux": "^7.1.16",
-    "@zeit/next-css": "^1.0.1",
-    "@zeit/next-less": "^1.0.1",
-    "@zeit/next-sass": "^1.0.1",
-    "algoliasearch": "^4.8.4",
-    "antd": "4.12.2",
-    "babel-plugin-import": "1.13.3",
-    "classnames": "^2.3.1",
-    "cross-env": "^7.0.3",
-    "current-git-branch": "^1.1.0",
-    "dotenv": "^8.2.0",
-    "esm": "^3.2.25",
-    "firebase": "^8.2.9",
-    "gray-matter": "4.0.2",
-    "highlight.js": "^10.5.0",
-    "less": "3.0.4",
-    "lodash": "^4.17.21",
-    "next": "10.0.1",
-    "null-loader": "2.0.0",
-    "postcss-preset-env": "^6.7.0",
-    "prop-types": "^15.7.2",
-    "raw-loader": "^4.0.2",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-icons": "^4.1.0",
-    "react-instantsearch": "^6.9.0",
-    "react-instantsearch-dom": "^6.9.0",
-    "react-markdown": "^6.0.2",
-    "react-redux": "^7.2.2",
-    "react-spring": "^9.1.2",
-    "react-visibility-sensor": "^5.1.1",
-    "reactfire": "^3.0.0-rc.0",
-    "redux": "^4.0.5",
-    "sass": "^1.32.8",
-    "tsconfig-paths": "^3.9.0"
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids"
   },
-  "devDependencies": {
-    "@babel/eslint-parser": "^7.12.13",
-    "@types/react": "^17.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.14.2",
-    "@typescript-eslint/parser": "^4.14.2",
-    "babel-plugin-inline-react-svg": "^1.1.2",
-    "eslint": "^7.19.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-airbnb-typescript": "^12.3.1",
-    "eslint-config-prettier": "^7.2.0",
-    "eslint-import-resolver-typescript": "^2.3.0",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-react": "^7.22.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-unicorn": "^27.0.0",
-    "husky": "^4.3.8",
-    "jest": "^26.6.3",
-    "lint-staged": "^10.5.4",
-    "prettier": "^2.2.1",
-    "redux-devtools": "^3.7.0",
-    "ts-node": "^9.1.1",
-    "typescript": "^4.1.3"
-  },
-  "browser": {
-    "fs": false,
-    "path": false
+  "engines": {
+    "node": "14.x"
```

Comment:
lets go >=16.x which is latest LTS

Reply:
Any reason not to move the 16.x?

Reply:
Oh I thought I replied to that comment last time. Anyway, Vercel currently doesn't support Node 16:
https://github.com/vercel/community/discussions/37

It will be supported in the next few weeks according to last comment so we can update when it's supported :) I first started with node 16 but deployment failed.

Reply:
Node 16 support added 5 days ago, we can probably give `"node": "16.x"` a try 🚀